src/common/metrics/registry.js

```diff
@@ -0,0 +1,120 @@
+const WavefrontSDKCounter = require('./counter').WavefrontSDKCounter;
```

Comment:
Generally use import syntax for these, but barring that, we use destructuring syntax: 
`const {WavefrontSDKCounter } = require("./counter");`